CHANGELOG.md

```diff
@@ -12,15 +12,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 -   Fix resetting of color range within color settings panel [#2877](https://github.com/MaibornWolff/codecharta/pull/2877)
 -   Fix resetting of colors in delta mode within color settings panel [#2873](https://github.com/MaibornWolff/codecharta/pull/2873)
 -   Restore global settings on page load again [#2878](https://github.com/MaibornWolff/codecharta/pull/2878)
+-   Fix windows standalone builds [#2881](https://github.com/MaibornWolff/codecharta/pull/2881)
 
 ### Chore 👨‍💻 👩‍💻
 
 -   Migrate color settings panel to Angular [#2873](https://github.com/MaibornWolff/codecharta/pull/2873)
 
-## [1.99.3] - 2022-06-28
```

Comment:
Would you please have a look at our .md files in gh-pages/_posts/release/? I think we should delete them as well.